.pre-commit-config.yaml

```diff
@@ -18,13 +18,15 @@ repos:
     - id: trailing-whitespace
       args: [--markdown-linebreak-ext=md]
 
-  # isort is a tool to sort and group import statements in Python files
-  # It works on files in-place
-  - repo: https://github.com/timothycrosley/isort
-    rev: 5.12.0
+  # ruff is a Python linter, incl. import sorter and formatter
+  # It works partly on files in-place
+  # More information can be found in its documentation:
+  # https://docs.astral.sh/ruff/
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.292
     hooks:
-    - id: isort
-      args: ["--profile", "black", "--filter-files", "--skip-gitignore"]
+      - id: ruff
+        args: [ --fix, --exit-non-zero-on-fix ]
```

Comment:
Why the extra indentation?
Could we also add a comment about why we chose these two arguments?

Reply:
There indeed shouldn't be extra indentation to follow the formatting of the rest of the file. For the understanding of the file it doesn't matter, but for conformity it does - will fix.